xml/bk_openstack_user.xml

```diff
@@ -520,17 +520,19 @@ top right of any page, you will see the <guimenu>Settings</guimenu> tab.</para>
 as an image, is a single file that contains a virtual disk that
 has a bootable operating system installed on it. Images are used
 to create virtual machine instances within the cloud. For information
-about creating image files, see the <link xlink:href="https://docs.openstack.org/image-guide/">OpenStack Virtual Machine
-Image Guide</link>.</para>
+about creating image files, see the <link
+xlink:href="https://docs.openstack.org/image-guide/">OpenStack Virtual Machine Image Guide</link>.</para>
       <para>Depending on your role, you may have permission to upload and manage
 virtual machine images. Operators might restrict the upload and
 management of images to cloud administrators or operators only. If you
 have the appropriate privileges, you can use the dashboard to upload and
 manage images in the admin project.</para>
       <note>
-        <para>You can also use the <command>openstack</command> and <command>glance</command>
-command-line clients or the Image service to manage images.
-For more information see .</para>
+        <para>You can also use the <command>openstack</command> and
+        <command>glance</command> command-line clients or the Image service to
+        manage images.  For more information see <link
```

Comment:
"You can also use the openstack or glance command-line clients, or the Image service to manage images. For more information..."
(remove the double space after "images." The other thing, the sentence read weirdly to me with the "and" between openstack and glance. I see what you mean though. Unsure how to reword without using "or" again. Thoughts?)